clients/metadata/metadata_cassandra_test.go

```diff
@@ -2056,48 +2106,80 @@ func (s *CassandraSuite) TestListConsumerGroups() {
 		testName = name
 	}
 
-	listReq.ConsumerGroupName = common.StringPtr(testName)
-	listRes, err = s.client.ListConsumerGroups(nil, listReq)
-	assert.Nil(err, "ListConsumerGroups failed to return results")
-	assert.Equal(1, len(listRes.GetConsumerGroups()), "ListConsumerGroups failed to return correct number of result")
-	assert.Equal(testName, listRes.GetConsumerGroups()[0].GetConsumerGroupName(), "Wrong consumer group returned")
+	testCGs := listCGs(&shared.ListConsumerGroupRequest{
+		DestinationPath:   common.StringPtr(dstPath),
+		ConsumerGroupName: common.StringPtr(testName),
+		Limit:             common.Int64Ptr(testPageSize),
+	})
+
+	assert.Equal(1, testCGs.Count(), "ListConsumerGroups failed to return correct number of result")
+	assert.True(testCGs.Contains(testName), "Wrong consumer group returned")
 
-	inputs := make([]*shared.ListConsumerGroupRequest, 0, 2)
-	inputs = append(inputs, &shared.ListConsumerGroupRequest{
+	assert.True(listCGs(&shared.ListConsumerGroupRequest{
 		DestinationPath: common.StringPtr(dstPath),
 		Limit:           common.Int64Ptr(testPageSize),
-	})
-	inputs = append(inputs, &shared.ListConsumerGroupRequest{
+	}).Equals(cgSet), "ListConsumerGroups did not return all CGs")
+
+	assert.True(listCGs(&shared.ListConsumerGroupRequest{
```

Comment:
Is this section duplicated from previous one?

Reply:
yes. removed.